packages/manager/modules/vps/src/upgrade/translations/Messages_fr_FR.json

```diff
@@ -31,5 +31,31 @@
   "vps_configuration_upgradevps_contracts_accept": "Veuillez accepter les contrats.",
   "vps_configuration_upgradevps_resume_selected_model": "Modèle sélectionné",
   "vps_configuration_upgradevps_resume_explain1": "En cliquant sur le bouton \"Commander\", un nouvel onglet s'ouvrira et vous serez automatiquement redirigé(e) vers l'interface de paiement.",
-  "vps_configuration_upgradevps_success": "Le bon de commande numéro {{ orderId }} a bien été créé. Afin de finaliser votre commande, veuillez suivre les instructions de paiement dans le nouvel onglet qui s'est ouvert. Si aucun nouvel onglet ne s'est ouvert, veuillez cliquer sur <a href=\"{{ url }}\" target=\"_blank\" rel=\"noopener\">ce lien</a>."
+  "vps_configuration_upgradevps_success": "Le bon de commande numéro {{ orderId }} a bien été créé. Afin de finaliser votre commande, veuillez suivre les instructions de paiement dans le nouvel onglet qui s'est ouvert. Si aucun nouvel onglet ne s'est ouvert, veuillez cliquer sur <a href=\"{{ url }}\" target=\"_blank\" rel=\"noopener\">ce lien</a>.",
```

Comment:
Do we do this or not ? As we talked about the other day, I don't like making the user think the Manager can be unsafe with a warning that we tried to open a tab

Reply:
Hmm in this case, the user action will directly open the tab, so, unless I'm wrong, the browser won't block it.

Reply:
Good point about the comma